### PR TITLE
Rename experimental benchmark job from "..._v2" to "..._gke_experimental"

### PR DESCRIPTION
**Pull request by jtattermusch**
Based on https://github.com/grpc/grpc/pull/27868 (will rebase after base PR is merged).

Currently the grpc_e2e_performance_v2 job has a confusing name and it's not clear it's the "experimental" job.
- renaming the corresponding shell script to make its experimental nature obvious
- create a new cfg file so I can create a new kokoro job with a more fitting name as a followup (after which I'm going to delete the _v2 job).

### CODE REVIEW

**Review by paulosjca — COMMENTED** [review]
Perhaps "experimental" instead of "experiment"? Otherwise it may seem to imply it is not permanent.

**Comment by jtattermusch** [comment]
> Perhaps "experimental" instead of "experiment"? Otherwise it may seem to imply it is not permanent.

Fair enough, but I chose the name to be consistent with other kokoro job names we have, so I'd rather stick with the name I proposed.

**Review by paulosjca — COMMENTED** [review]
Please update references in tools/run_tests/performance/README.md as well.

**Comment by jtattermusch** [comment]
> Please update references in tools/run_tests/performance/README.md as well.

Good catch, done.

**Comment by jtattermusch** [comment]
This should be ready for approval and merging?

File: tools/run_tests/performance/README.md [hunk 1]
```diff
@@ -299,11 +299,11 @@ For more information, see the
 in [grpc/test-infra].
 
 For usage examples, see the continuous integration setup defined in
-[grpc_e2e_performance_gke.sh] and [grpc_e2e_performance_v2.sh].
+[grpc_e2e_performance_gke.sh] and [grpc_e2e_performance_gke_experiment.sh].
 
 [grpc/test-infra]: https://github.com/grpc/test-infra
 [grpc_e2e_performance_gke.sh]: ../../internal_ci/linux/grpc_e2e_performance_gke.sh
-[grpc_e2e_performance_v2.sh]: ../../internal_ci/linux/grpc_e2e_performance_v2.sh
+[grpc_e2e_performance_gke_experiment.sh]: ../../internal_ci/linux/grpc_e2e_performance_gke_experiment.sh.sh
```
Comment (paulosjca) [inline tools/run_tests/performance/README.md:306]: Remove duplicate `.sh`.

Comment (jtattermusch) [reply tools/run_tests/performance/README.md]: oops, done.

**Review by paulosjca — APPROVED** [review]

**Pull request merged**